docs/deployment/worker.md

```diff
@@ -72,6 +72,30 @@ Example submission script:
 The worker will try to automatically detect that it is started under a PBS/Slurm job, but you can also explicitly pass
 the option `--manager <pbs/slurm>` to tell the worker that it should expect a specific environment.
 
+#### Deploying a worker using SSH
+
+If you have an OpenSSH-compatible `ssh` binary available in your environment, HQ can deploy workers to a set of hostnames using the `deploy-ssh` command:
+
+```bash
+$ hq worker deploy-ssh <nodefile> <worker-args>
+```
+
+For example, if we had the following content stored in file called `nodefile.txt`:
```

Comment:
The documentation should probably explicitly say that `node1` is a hostname. I am not sure that it is a common knowledge what "nodefile" means.

Reply:
Also maybe calling it `hostfile` would be more descriptive.

Reply:
Changed documentation and renamed it to `hostfile`.